CONTRIBUTING.md

```diff
@@ -137,18 +137,21 @@ If we accept the PR, a [maintainer](MAINTAINERS.md) will merge your change and u
 
 If we reject the PR, we will close the pull request with a comment explaining why. This decision isn't always final: if you feel we have misunderstood your intended change or otherwise think that we should reconsider then please continue the conversation with a comment on the PR and we'll do our best to address any further points you raise.
 
-## Feature Branches
+## Using Feature Branches
 
-For long running features that need collaboration between multiple members of the community, we use feature branches. We use a hybrid of the feature branch development model and the continuous integration development model. Maintainers of a repository can create a feature branch, a corresponding label associated with the feature and a meta issue associated with the feature. Each feature branch will have the following safeguards:
+Our recommend approach for development is doing frequent small PR merges to main. This lets us catch integration issues earlier and makes the development visible to everyone.  It's okay if it's not the complete feature, as long as the PR won’t break a build or any existing functionality at least. 
```

Comment:
It will make sure PRs are quick and easy to review.

Reply:
Added!